src/Bicep.Core/Configuration/ConfigHelper.cs

```diff
@@ -50,7 +50,7 @@ private IConfigurationRoot BuildConfig(string? localFolder)
                 // last added json settings take precedent - add local settings last
                 if (DiscoverLocalConfigurationFile(localFolder) is string localConfig)
                 {
-                    configBuilder.AddJsonFile(localConfig, true, true);
+                    configBuilder.AddJsonFile(localConfig, true, false);
```

Comment:
Use named parameters if you want to be a **cool** kid. All the knowledge of what these parameters do is buried 😆

Reply:
I couldn't resist...

Reply:
😄

This is something that we should set up a linter for and enforce it at some point.